reflex/utils/prerequisites.py

```diff
@@ -432,13 +432,23 @@ def install_frontend_packages(packages: set[str]):
     processes.show_status("Installing base frontend packages", process)
 
     config = get_config()
-    if config.tailwind is not None and "plugins" in config.tailwind:
+    if config.tailwind is not None:
+        # install tailwind
         process = processes.new_process(
-            [get_install_package_manager(), "add", *config.tailwind["plugins"]],
+            [get_install_package_manager(), "add", constants.TAILWIND_VERSION, "-d"],
```

Comment:
```suggestion
            [get_install_package_manager(), "add", "-d", constants.TAILWIND_VERSION, *config.tailwind.get("plugins", [])],
```

does this work? seems a little more compact.

Reply:
Yeah this works :)